### PR TITLE
002 view job spec

### PR DESCRIPTION
**Pull request by MihneaBocaKainos**

### CODE REVIEW

File: src/main/java/com/kainos/ea/util/DatabaseConnector.java [hunk 1]
```diff
@@ -33,10 +34,26 @@ public Connection getConnection() throws DatabaseException, SQLException {
...
+
+
+    public static void closeConnection()  {
+        try{
+            if (conn != null) {
+                conn.close();
+                conn = null;
+            }
+        }catch (SQLException e ){
+            // do something
```
Comment (hellofadude) [inline src/main/java/com/kainos/ea/util/DatabaseConnector.java:56]: remove unnecessary comment

File: src/main/java/com/kainos/ea/dao/EmployeeDao.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.kainos.ea.util.DatabaseConnector.closeConnection;
+import static com.kainos.ea.util.DatabaseConnector.getConnection;
```
Comment (hellofadude) [inline src/main/java/com/kainos/ea/dao/EmployeeDao.java:13]: Extra line after import statement and before class definition for clarity

File: src/main/java/com/kainos/ea/model/JobRole.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package com.kainos.ea.model;
+
+
```
Comment (hellofadude) [inline src/main/java/com/kainos/ea/model/JobRole.java:3]: Remove extra line

File: src/main/java/com/kainos/ea/model/JobRole.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+@Getter @Setter
+public class JobRole {
+
+    protected int jobid;
+
+
+    protected String jobName;
+
+
```
Comment (hellofadude) [inline src/main/java/com/kainos/ea/model/JobRole.java:17]: remove extra line

File: src/main/java/com/kainos/ea/model/JobRole.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter @Setter
+public class JobRole {
+
+    protected int jobid;
+
+
```
Comment (hellofadude) [inline src/main/java/com/kainos/ea/model/JobRole.java:14]: remove extra line

**Review by hellofadude — APPROVED** [review]

**Review by asadzafark — APPROVED** [review]

**Pull request merged**